src/apis/document_apis.jsx

```diff
@@ -0,0 +1,24 @@
+import AllDocs from "@/apis/test_documents";
+
+/*
+    Gets all information on all docs
+*/
+function GetAllDocs() {
+    return AllDocs
+}
+
+/*
+    Gets ids, name, and dateLastModified (and other metadata) info about all docs
+*/
+function GetAllDocIdsAndMetadata() {
```

Comment:
Hmmm I think this doc metadata API shouldn't return the whole document body, otherwise our network times will suck ass. so no need to delete `body`? Might also be nice to just annotate what format you expect as the result of each of these APIs in a comment for when we implement them.

Also, just a nit but let's call the folder `util`? Just so the import path works out a little cleaner, otherwise the class and folder names are a little redundant.

Reply:
As per conversation: Only change needed is renaming folder to `util`